cmd/storj-sim/network.go

```diff
@@ -217,12 +229,21 @@ func newNetwork(flags *Flags) (*Processes, error) {
 	versioncontrol.ExecBefore["run"] = func(process *Process) error {
 		return readConfigString(&versioncontrol.Address, versioncontrol.Directory, "address")
 	}
-
 	// gateway must wait for the versioncontrol to start up
 
+	// start a test redis server
+	rootpath := flags.Redis
+	if flags.Redis == "testServer" {
+		address, cleanup, err := redisserver.Start()
```

Comment:
each satellite should probably have a separate redis instance

Reply:
ok